tools/gradle/BUILDING.md

```diff
@@ -0,0 +1,83 @@
+# Building Ice Slice Tools Gradle Plugin
```

Comment:
Building the Ice Slice Tools Gradle Plugin

or maybe "the Ice Slice Tools for Gradle"?

Reply:
What about a simpler:  Slice Tools for Ice. The Gradle plugin part moves to the description.